docs/nodes/nodes-library/trigger-nodes/TypeformTrigger/README.md

```diff
@@ -24,9 +27,77 @@ The final workflow should look like the following image.
 
 ### 1. Typeform Trigger node
 
+This node will trigger the workflow when a form response is submitted.
+
+This example workflow uses a Typeform to collect name and email address. Below are the questions and their question type used in the form.
+
+|Question | Type  |
+|---------|-------|
+|Let's start with your name | Short Text |
+|What's your email address? | Email |
+
+If you haven't, create a form similar to this [form](https://n8ndocsburner.typeform.com/to/dpr2kxSL).
```

Comment:
@harshil1712 Should it be similar to or exactly same? From a first glance, it seems that the expression and the example workflow will fail if their questions aren't exactly the same.